ios/Trifle/Sources/Trifle.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Trifle.swift
+//  Trifle
+//
+
+import Foundation
+
+public class Trifle {
+    
+    public static let version = "0.1.3"
```

Comment:
is there a reason this differs from the rest of the lib?  Also, how do we feel about divergence between iOS and android/jvm in general (versioning-wise)

Reply:
I was going to bump podspec to 0.1.3 as the last PR before the tag release. Just preempting it

Reply:
ack